nixos/modules/services/hardware/logiops.nix

```diff
@@ -0,0 +1,144 @@
+{ config, lib, pkgs, ... }:
+
+let
+  inherit (lib) mkEnableOption mkIf mkOption types;
+  cfg = config.services.logiops;
+  format = pkgs.formats.libconfig { };
+in {
+  options.services.logiops = {
+    enable = mkEnableOption
+      "logiops, an unofficial userspace driver for HID++ Logitech devices";
+
```

Comment:
I forgot to mention this: can you please add a `package = mkPackageOption …` here and use it instead of `pkgs.logiops`? Thanks.